Framework/Beamline/test/ComponentInfoTest.h

```diff
@@ -66,7 +66,7 @@ makeFlatTree(PosVec detPositions, RotVec detRotations) {
 
   std::vector<size_t> children(detPositions.size());
   std::iota(children.begin(), children.end(), 0);
-  auto instrumentTree =
+  auto assemblyImmediateChildren =
```

Comment:
I like the `ImmedateChildren` part, but as this is also filled in for non-assemblies (i.e. regular components), maybe we should change the `assembly` part of the name to `component`?